lib/holdings.rb

```diff
@@ -59,8 +59,9 @@ def mhld
     end
   end
 
-  def as_json(*)
```

Comment:
Yes, I think rails might (or may have) passed an implicit parameter when doing something like `render json: myObject` (where that would effectively call `myObject.as_json(root: true)` or something like that).  I think that parameter has an affect on ActiveRecord objects (where it's typically used) but in this case we don't want to do anything different.